plugin/pkg/scheduler/testing/fake_lister.go

```diff
@@ -40,13 +33,6 @@ func (f FakeNodeLister) List() ([]*v1.Node, error) {
 	return f, nil
 }
 
```

Comment:
Just a nit (and not a blocker), could be worth to introduce `var _ NodeLister = &FakeNodeLister{}` and alike assignments to have the go compiler implicitly check all the ``Fake*Lister`` implements corresponding interfaces.